apis/kueue/v1alpha2/workload_types.go

```diff
@@ -113,6 +113,10 @@ const (
 	// WorkloadFinished means that the workload associated to the
 	// ResourceClaim finished running (failed or succeeded).
 	WorkloadFinished = "Finished"
+
+	// WorkloadPodsReady means that the first batch of Job pods (up to the
+	// specified parallelism and completions) is either ready or succeeded.
```

Comment:
That's specific to batch.Job, whereas Workload is supposed to be generic.

Maybe you can say that at least `.spec.podSets[*].replicas` Pods are ready or have succeeded.

Reply:
Adjusted